Embed WebVI into Content/Readme.md

```diff
@@ -122,17 +140,14 @@ In order for the WebVI to continue to run in the new page these two custom eleme
 The `ni-virtual-instrument` custom element needs no modification after it has been emitted into `Main.html` from LabVIEW. The `vireo-source` property `web-application` must be updated to reflect the relative path between `index.html` and `Main.via.txt`.
 
 ```
```

Comment:
can add an html specifier to get syntax highlighting

```
  ```html
  <my html>
```

looks like this:

```html
<ni-web-application engine="VIREO" location="BROWSER" vireo-source="WebVI/Builds/Web%20Server/Configuration1/WebApp/Main.via.txt"><ni-virtual-instrument vi-name="Main.gviweb"></ni-virtual-instrument></ni-web-application>
```

instead of 

```
<ni-web-application engine="VIREO" location="BROWSER" vireo-source="WebVI/Builds/Web%20Server/Configuration1/WebApp/Main.via.txt"><ni-virtual-instrument vi-name="Main.gviweb"></ni-virtual-instrument></ni-web-application>
```

The github syntax highlighter is not completely on point with custom elements, but it's still a bit better to read.